### PR TITLE
Init OWNERS and pr template

### PR DESCRIPTION
**Pull request by yutongz**
Add files for mungegithub. 

Here is a refer: https://github.com/istio/mixer/pull/1024

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+reviewers:
```
Comment (ldemailly) [inline OWNERS]: list seems a bit too short / bottleneck-y

Comment (yutongz) [inline OWNERS]: I agree, and I think the best part of this OWNERS system is that you can specify people in a specific area, which means we can put more people in the package he/she is mainly working on. The root OWNERS might be more suitable for people taking charge of the whole repo like Kuat and rshriram.

**Comment by yutongz** [comment]
@rshriram sure thing

File: OWNERS (hunk 1 above)
Comment (rshriram) [inline OWNERS]: In that case, put @GregHanson for consul, @ijsnellf for eureka, @ayj for kube, in addition to me and Kuat. I am probably missing more folks but these should be a good coverage for starters and we can add people as time progresses.

Comment (yutongz) [inline OWNERS]: Sounds good.

Comment (ldemailly) [inline OWNERS]: I'd add andra too

**Comment by yutongz** [comment]
Can anyone help me merge it and I will deploy mungegithub right way.

**Pull request merged**